client/analytics/report/stock/table.js

```diff
@@ -25,6 +26,29 @@ class StockReportTable extends Component {
 		this.getHeadersContent = this.getHeadersContent.bind( this );
 		this.getRowsContent = this.getRowsContent.bind( this );
 		this.getSummary = this.getSummary.bind( this );
+		this.customizeStockStatusForCSVDownload();
+	}
+
+	customizeStockStatusForCSVDownload() {
+		addFilter(
+			'woocommerce_admin_report_table',
```

Comment:
This filter gets passed the result of the `getRowsContent()` call. Can we just modify this line instead of using a filter? https://github.com/woocommerce/woocommerce-admin/blob/2bc265c137c45b1cf33b797f3038943d2c3845b5/client/analytics/report/stock/table.js#L146

Reply:
Thank you @jeffstieler. That's a simpler solution 👍 Updated in https://github.com/woocommerce/woocommerce-admin/pull/7284/commits/86c75017fb2241d9f405a1fe804f88e5d0578cf9